python/scripts/meta.json

```diff
@@ -171,6 +171,178 @@
     ],
     "replace_table": false
   },
+  "census_tract_to_neighborhood_cluster": {
+    "cleaner": "CensusTractToNeighborhoodClusterCleaner",
+    "fields": [
+      {
+        "display_name": "Census tract",
+        "display_text": "Census tract",
+        "required_in_source": true,
+        "source_name": "Geo2010",
+        "sql_name": "census_tract",
+        "type": "text"
+      },
+      {
+        "display_name": "Neighborhood cluster description",
+        "display_text": "Neighborhood cluster description",
+        "required_in_source": true,
+        "source_name": "Cluster_tr2000",
+        "sql_name": "cluster_desc",
+        "type": "text"
+      },
+      {
+        "display_name": "Cluster tract population",
+        "display_text": "Neighborhood Cluster-Tract piece population",
+        "required_in_source": true,
+        "source_name": "Pop",
+        "sql_name": "cluster_tract_population",
+        "type": "integer"
+      },
+      {
+        "display_name": "Tract pieces",
+        "display_text": "Tract pieces split by Neighborhood Cluster",
+        "required_in_source": true,
+        "source_name": "Pieces_tr10",
+        "sql_name": "num_tract_pieces_in_cluster",
+        "type": "integer"
+      },
+      {
+        "display_name": "Tract population",
+        "display_text": "Tract population",
+        "required_in_source": true,
+        "source_name": "Pop_tr10",
+        "sql_name": "tract_population",
+        "type": "integer"
+      },
+      {
+        "display_name": "Cluster pieces",
+        "display_text": "Neighborhood Cluster pieces split by Tract",
+        "required_in_source": true,
+        "source_name": "Pieces_cltr00",
+        "sql_name": "num_cluster_pieces_in_tract",
+        "type": "integer"
+      },
+      {
+        "display_name": "Cluster population",
+        "display_text": "Cluster population",
+        "required_in_source": true,
+        "source_name": "Pop_cltr00",
+        "sql_name": "cluster_population",
+        "type": "integer"
+      },
+      {
+        "display_name": "Population weight in tract",
+        "display_text": "Population weight for counts (Pop / Pop_tr10)",
+        "required_in_source": true,
+        "source_name": "Popwt",
+        "sql_name": "population_weight_in_tract",
```

Comment:
I think it will be clearer to future users (both developers and anyone accessing the API) to name this `population_weight_counts` and the other one `population _weight_proportions`.